### DISCUSSION TITLE
充值的专业版升级后变免费版，求蓝灯团队解决

### DISCUSSION THREAD

**Discussion by BLUE-WJP** [getlantern/lantern-forum-cn#430] [General]
邮箱账号：965003277@qq.com2024年3月份刚充的一年会员，现在变免费版了

**Comment by BLUE-WJP** [comment]


**Reply by BLUE-WJP** [reply]
![2be48336eebfafbd594ee8c70406c3d](https://github.com/getlantern/lantern-forum-cn/assets/144672465/a6cf01d1-03b7-4352-b4d4-17e62fbc48ae)

**Reply by MaxMKMax** [reply]
@BLUE-WJP 您好. 我已经报您提交问题上去了. 过几天请查看邮箱或GitHub. 谢谢.

**Comment by BLUE-WJP** [comment]
好的，谢谢

**Reply by MaxMKMax** [reply]
@BLUE-WJP 不客气呢. 祝您有美好的周末!

**Comment by Eddiekazsh** [comment]
同样的问题，去年的专业版付费用户，邮箱soflash@qq.com，麻烦帮忙解决 @Mike777Khoo
![lantern-receipt](https://github.com/getlantern/lantern-forum-cn/assets/166973291/f258c9d4-b4e8-4046-8d6c-48f205cbcf1e)

**Reply by MaxMKMax** [reply]
@Eddiekazsh 嗯嗯. 好的. 没问题. 谢谢您支持蓝灯!

**Reply by Eddiekazsh** [reply]
@Mike777Khoo 请尽快解决我的账号问题，谢谢！

**Comment by BLUE-WJP** [comment]
一半要多久我能收到解决问题的那个邀请码，邮件今天还没有收到





---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年4月14日(周日) 晚上10:29
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 充值的专业版升级后变免费版，求蓝灯团队解决 (Discussion #430)




 
@Eddiekazsh 嗯嗯. 好的. 没问题. 谢谢您支持蓝灯!

**Reply by MaxMKMax** [reply]
@BLUE-WJP 您好. 支付团队说您不是向我们直接购买的. 您是不是在别的网站购买的? 您方便给我发那个网站的链接吗? 因为我们不确定是不是合作伙伴的网站.

**Comment by BLUE-WJP** [comment]
***@***.***这个邮箱就是蓝灯账号，我充值支付的是这个账号的，不是我发问题邮件的这个账号，充值支付信息截屏我都发在帖子里了的，你们没有核对吗?

---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年4月16日(周二) 晚上11:06
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 充值的专业版升级后变免费版，求蓝灯团队解决 (Discussion #430)




 
@BLUE-WJP 您好. 支付团队说您不是向我们直接购买的. 您是不是在别的网站购买的? 您方便给我发那个网站的链接吗? 因为我们不确定是不是合作伙伴的网站.

**Reply by MaxMKMax** [reply]
@BLUE-WJP 不是. 您误会了我的意思. 我们支付团队说您的单据不是我们公司的. app确实是我们的. 但我们也有经销商. 估计您是在别的地方购买的. 所以想问您在哪里买的?

**Comment by BLUE-WJP** [comment]
这个是交易信息，你看下对不对



---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年4月17日(周三) 凌晨4:02
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 充值的专业版升级后变免费版，求蓝灯团队解决 (Discussion #430)




 
@BLUE-WJP 不是. 您误会了我的意思. 我们支付团队说您的单据不是我们公司的. app的却是我们的. 但我们也有经销商. 估计您是在别的地方购买的. 所以想问您在哪里买的?

**Reply by MaxMKMax** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/54c185ee-baf9-4489-a471-4d2e9e530cf3)
您好. 您的截图或信息被GitHub屏蔽了. 请您发到我的邮件吧. 然后在这里告诉我您的邮箱是什么. 我帮您查看. 谢谢!

**Comment by BLUE-WJP** [comment]
---原始邮件---
发件人: "Mike ***@***.***&gt;
发送时间: 2024年5月8日(周三) 凌晨1:11
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [getlantern/lantern-forum-cn] 充值的专业版升级后变免费版，求蓝灯团队解决 (Discussion #430)




 
image.png (view on web)
 您好. 您的截图或信息被GitHub屏蔽了. 请您发到我的邮件吧. 然后在这里告诉我您的邮箱是什么. 我帮您查看. 谢谢!
